dev/com.ibm.ws.sipcontainer/src/com/ibm/ws/sip/stack/transport/chfw/GenericEndpointImpl.java

```diff
@@ -10,6 +10,7 @@
  *******************************************************************************/
 package com.ibm.ws.sip.stack.transport.chfw;
 
+import java.net.*;
```

Comment:
Could you update it so that only the required classes are imported?
```
import java.net.InetAddress;
import java.net.Inet6Address;
import java.net.Inet4Address;
```